core/src/com/unciv/logic/map/MapGenerator.kt

```diff
@@ -653,6 +665,38 @@ class MapGenerator(val ruleset: Ruleset) {
             return Perlin.noise3d(worldCoords.x.toDouble(), worldCoords.y.toDouble(), seed, nOctaves, persistence, lacunarity, scale)
         }
 
+        private fun getDistortedPerlinNoise(tile: TileInfo, seed: Double,
+                                   nOctaves: Int = 6,
+                                   persistence: Double = 0.5,
+                                   lacunarity: Double = 2.05,
+                                   scale: Double = 20.0): Double {
+            val worldCoords = HexMath.hex2WorldCoords(tile.position)
+
+            var noise = Perlin.noise3d(worldCoords.x.toDouble()+100.5, worldCoords.y.toDouble()+100.5, seed+100.5, 1, persistence, lacunarity, scale*20.0)
```

Comment:
z axis is modified to get noise variety but it might not be the most elegant solution.